modules/unsupported/process/src/main/java/org/geotools/process/function/ProcessFunctionFactory.java

```diff
@@ -137,32 +135,37 @@ private synchronized void init() {
             processToFunction = new HashMap<Name,FunctionName>();
             functionNames = new ArrayList<FunctionName>();
             for (ProcessFactory factory : factories) {
+                if( !factory.isAvailable() ) continue;
```

Comment:
Not a stopped, you might just ignore the comment, but this does not align with our coding conventions, it should have been:

````
if(!factory.isAvailable()) {
  continue;
}